src/main/java/edu/hm/hafner/echarts/UpperLabel.java

```diff
@@ -0,0 +1,36 @@
+package edu.hm.hafner.echarts;
+
+/**
+ * Label style for an item with children.
+ * <p>
+ * This class will be automatically converted to a JSON object.
+ * </p>
+ *
+ * @author Florian Orendi
+ */
+public class UpperLabel {
```

Comment:
Why is this a copy of the `Label` class? Can't we simply have a property `upperLabel` of the `Label` in class `TreeMapNode`?